### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by SanjanaSogimatt**
## 📋 Description
Updating readme

## Summary by CodeRabbit

- **New Features**
	- Introduced a new "AMRIT" health technology platform in the README.
	- Added a new "Credits" page to acknowledge contributors.
	- Enhanced footer with a link to the credits section.

- **Bug Fixes**
	- Corrected spelling of "gallery" in the about page comments.

- **Documentation**
	- Updated README to reflect changes and provide clearer instructions for running the project locally.
	- Added detailed contributor information in the new credits markdown file.
	- Revised contribution guidelines to encourage community involvement.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,123 +1,48 @@
 <h1 align=center>Pinwheel Astro</h1>
```
Comment (drtechie) [inline README.md:1]: Let's change this title too.
Give credits in the readme and links to original forked repo.

File: README.md [hunk 2]
```diff
@@ -1,123 +1,48 @@
...
-<p align=center>Pinwheel is a free multipurpose SaaS theme built with Astro, TailwindCSS and TypeScript with 17+ pre-designed pages.</p>
-<h2 align="center"><a target="_blank" href="https://pinwheel-astro.vercel.app/" rel="nofollow"> 👀 Demo</a> | <a  target="_blank" href="https://pagespeed.web.dev/analysis/https-pinwheel-astro-vercel-app/kmaxqwa7rx?form_factor=desktop"> Page Speed (95+) 🚀 </a>
-</h2>
+# AMRIT (Accessible Medical Records via Integrated Technologies)
```
Comment (drtechie) [inline README.md:2]: Something is off with this readme. The titles and descriptions don't look aligned when viewed in GitHub.
https://github.com/PSMRI/AMRIT-Website/tree/update-readme

File: src/content/credits/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Acknowledgments
+subtitle: We extend our heartfelt gratitude to everyone who contributed to the development of this project. Your expertise, creativity, and support have made this endeavor a success.
+team: 
+    title: Contributers
+    list: 
+        - name: Sanjana Sogimatt
+          title: Commuinty maintainer
```
Comment (drtechie) [inline src/content/credits/index.md:8]: Community typo

File: src/content/credits/index.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+title: Acknowledgments
+subtitle: We extend our heartfelt gratitude to everyone who contributed to the development of this project. Your expertise, creativity, and support have made this endeavor a success.
+team: 
+    title: Contributers
```
Comment (drtechie) [inline src/content/credits/index.md:5]: Contributors typo

File: src/content/credits/index.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    list: 
+        - name: Sanjana Sogimatt
+          title: Commuinty maintainer
+          github: "https://github.com/SanjanaSogimatt"
+          linkedin: "https://www.linkedin.com/in/sanjana-sogimatt-764a931ba/"
+          image: "/images/credits/sanjana.webp"
+        - name: Adarsh Jena
+          title: Contributer
```
Comment (drtechie) [inline src/content/credits/index.md:13]: Contributor typo

File: src/layouts/partials/Footer.astro [hunk 6]
```diff
@@ -6,7 +6,7 @@ import menu from "@/config/menu.json";
 import social from "@/config/social.json";
 import { markdownify } from "@/lib/utils/textConverter";
 const { footer } = menu;
-const { copyright, location, email, phone, footer_description } = config.params;
+const { copyright, location, email, phone, footer_description,credits } = config.params;
```
Comment (drtechie) [inline src/layouts/partials/Footer.astro:9]: space after comma.

File: src/layouts/partials/Footer.astro [hunk 7]
```diff
@@ -60,6 +60,10 @@ const { copyright, location, email, phone, footer_description } = config.params;
       class="footer-copyright mx-auto border-t border-border pb-10 pt-7 text-center"
     >
       <p class="content text-text" set:html={markdownify(copyright)} />
+      <div>
+        <a href={`${credits}`} class="underline">Made by ❤️ Amrit Community</a>
```
Comment (drtechie) [inline src/layouts/partials/Footer.astro:64]: Made with ❤️ by AMRIT Community?

Comment (drtechie) [reply src/layouts/partials/Footer.astro:64]: Maybe add a little bit of top padding.

File: src/pages/credits.astro [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+            credits.data.team.list.map((content: any) => (
+              <div class="flex flex-col bg-white min-h-[250px] rounded-xl shadow-md p-5 pb-8 text-center items-center justify-center w-full max-w-lg">
+                <div class="w-full flex justify-center">
+                    <Image
+                      height={300}
+                      width={200}
+                      class="object-cover rounded-2xl"
+                      src={content.image}
+                      alt=""
+                    />
```
[the remark this answers is not in the document]
Comment (drtechie) [reply src/pages/credits.astro:29]: @SanjanaSogimatt you can put the contributor name here.

**Comment by drtechie** [comment]
We can also consider adding credits for original theme

File: src/pages/credits.astro (hunk 8 above)
[the remark this answers is not in the document]
Comment (SanjanaSogimatt) [reply src/pages/credits.astro:29]: Okay

File: README.md (hunk 1 above)
Comment (SanjanaSogimatt) [reply README.md:1]: Understood

File: README.md (hunk 2 above)
Comment (SanjanaSogimatt) [reply README.md:2]: Okay will fix that

File: README.md [hunk 9]
```diff
@@ -1,123 +1,60 @@
...
 
-<!-- small description -->
+1. Clone the repository:
 
-<!-- key features -->
-## 📌 Key Features
+   ```bash
+   git clone https://github.com/AMRIT-Website/AMRIT.git
+   cd AMRIT
+   ```
```
[the remark this answers is not in the document]
Comment (drtechie) [reply README.md:14]: @SanjanaSogimatt the repo URL needs correction.

**Pull request merged**